test/services/AuditServiceSpec.scala

```diff
@@ -50,55 +54,56 @@ class AuditServiceSpec extends BaseSpec {
         "clientPort" -> "-",
         "transactionName" -> name
       )
-    val approvalEvent = ApprovedForPublishingEvent(PID, processID, processTitle)
-    val factCheckEvent = FactCheckSubmissionEvent(PID, processID, processTitle)
-    val readyForPubEvent = ReadyForPublishingEvent(PID, processID, processTitle)
+    val auditInfo: AuditInfo = AuditInfo(PID, processID, processTitle, processVersion, ocelotAuthor, ocelotLastUpdate, ocelotVersion)
+    val twoEyeReviewCompleteEvent: TwoEyeReviewCompleteEvent = TwoEyeReviewCompleteEvent(auditInfo)
+    val factCheckCompleteEvent: FactCheckCompleteEvent = FactCheckCompleteEvent(auditInfo)
+    val publishedEvent: PublishedEvent = PublishedEvent(auditInfo)
   }
 
   "The Audit service" should {
 
-    "Accept an ApprovedForPublishingEvent object" in new Test {
-      val details = Json.toJson(approvalEvent)
-      val path = Some("/guidance/scratch")
-      val extendedEvent = ExtendedDataEvent(
+    "Accept an TwoEyeReviewCompleteEvent object" in new Test {
+      val details: JsValue = Json.toJson(twoEyeReviewCompleteEvent)
+      val path: Option[String] = Some("/guidance/scratch")
+      val extendedEvent: ExtendedDataEvent = ExtendedDataEvent(
         "manage-external-guidance-frontend",
-        "approvedForPublishing",
+        "2iReviewCompleted",
         eventUUID,
-        tagsData("approvedForPublishing", path),
+        tagsData("2iReviewCompleted", path),
         details,
         submissionTime
       )
       MockAuditConnector.sendExtendedEvent(extendedEvent)
```

Comment:
It would be good if we could verify the argument input to the mock connector, but at present I'm not certain how to do this.

Reply:
I'm not sure how we verify this eaither - only thing I could do was check the log to ensure the right data/events was being sent when processes moved on state